lib/axiom/fs/base/execute_context.js

```diff
@@ -213,6 +205,27 @@ ExecuteContext.prototype.setStdio = function(stdio) {
 };
 
 /**
+ * @return {!Stdio}
+ */
+ExecuteContext.prototype.getStdio = function() {
```

Comment:
I don't understand why this is needed. If you had left this.stdio = stdio in the constructor, this function would not be needed, right?